air-quality-ui/e2e/summary.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { expect, test } from "@playwright/test";
+import { apiForecast, apiSummary } from "./mocked_api.ts";
+
+test("Mocked response breadcrumb", async ({ page }) => {
+  await page.route("*/**/air-pollutant/forecast*", async (route) => {
+    await route.fulfill({ json: apiForecast });
+  });
+  await page.route(
+    "*/**/air-pollutant/measurements/summary*",
+    async (route) => {
+      await route.fulfill({ json: apiSummary });
+    }
+  );
+  await page.goto("http://localhost:5173/city/summary");
```

Comment:
Can we pull this out into a environment variable or config item or something? We'll need to create a few playwright tests I imagine so having it in one easy to change place would be good.

Reply:
Makes sense, I can make this a baseURL inside playwright.config file for less repetition of code 😎